ui/component/fileRender/view.jsx

```diff
@@ -80,12 +81,23 @@ class FileRender extends React.PureComponent<Props> {
   }
 
   renderViewer() {
-    const { mediaType, currentTheme, claim, contentType, downloadPath, fileName, streamingUrl, uri } = this.props;
+    const {
+      mediaType,
+      currentTheme,
+      claim,
+      contentType,
+      downloadPath,
+      fileName,
+      streamingUrl,
+      embedUrl,
+      uri,
+    } = this.props;
     const fileType = fileName && path.extname(fileName).substring(1);
+    const streamUrl = embedUrl || streamingUrl;
 
     // Ideally the lbrytv api server would just replace the streaming_url returned by the sdk so we don't need this check
     // https://github.com/lbryio/lbrytv/issues/51
-    const source = IS_WEB ? generateStreamUrl(claim.name, claim.claim_id) : streamingUrl;
+    const source = IS_WEB ? generateStreamUrl(claim.name, claim.claim_id) : streamUrl;
```

Comment:
Why is `streamUrl` including `embedUrl`? That should only be used if `IS_WEB` is true, which would never be the case here.